source/puppet/5.3/hiera_automatic.md

```diff
@@ -93,37 +96,39 @@ lookup( { 'name'  => 'users',
-* `<OPTIONS HASH>` (Hash) - Alternate way to set the arguments above, plus some less common additional options. If you pass an options hash, you can’t combine it with any regular arguments (except `<NAME>`). An options hash can have the following keys:
-	* `'name'` - Same as `<NAME>` (argument 1). You can pass this as an argument or in the hash, but not both.
-	* `'value_type'` - Same as `<VALUE TYPE>`.
-	* `'merge'` - Same as `<MERGE BEHAVIOR>`.
-	* `'default_value'` - Same as `<DEFAULT VALUE>` .
-	* `'default_values_hash'` (Hash) - A hash of lookup keys and default values. If Hiera can’t find a normal value, it will check this hash for the requested key before giving up. You can combine this with `default_value` or a lambda, which will be used if the key isn’t present in this hash. Defaults to an empty hash.
-	* `'override'` (Hash) - A hash of lookup keys and override values. Puppet will check for the requested key in the overrides hash first. If found, it returns that value as the final value, ignoring merge behavior. Defaults to an empty hash.
-	* `lookup` - can take a lambda, which must accept a single parameter. This is yet another way to set a default value for the lookup; if no results are found, Puppet will pass the requested key to the lambda and use its result as the default value.
-
-Related topics: [Data type][data_type]
-
-{:.concept} 
+{:.reference}
+## Arguments accepted by lookup
+
+You must provide the key's name. The other arguments are optional.
+
+-   `<NAME>` (String or Array) - The name of the key to look up.
+This can also be an array of keys. If Hiera doesn't find anything for the first key, it tries with the subsequent ones, only resorting to a default value if none of them succeed.
+-   `<VALUE TYPE>` (data Type) - A data type that must match the retrieved value; if not, the lookup (and catalog compilation) will fail. Defaults to `Data`  which accepts any normal value.
+-   `<MERGE BEHAVIOR>` (String or Hash; see “Merge Behaviors”) -  Whether and how to combine multiple values. If present, this overrides any merge behavior specified in the data sources. Defaults to no value; Hiera will use merge behavior from the data sources if present, and will otherwise do a first-found lookup.
+-   `<DEFAULT VALUE>` (any normal value) - If present, lookup returns this when it can't find a normal value. Default values are never merged with found values. Like a normal value, the default must match the value type. Defaults to no value; if Hiera can't find a normal value, the lookup (and compilation) will fail.
+-   `<OPTIONS HASH>` (Hash) - Alternate way to set the arguments above, plus some less common additional options. If you pass an options hash, you can't combine it with any regular arguments (except `<NAME>`). An options hash can have the following keys:
+    -   `'name'` - Same as `<NAME>` (argument 1). You can pass this as an argument or in the hash, but not both.
```

Comment:
Note the indentation here. Four spaces, then the list character (`-`), then three spaces. Each level of indentation consistently uses four characters.